charts/posthog/templates/pgbouncer-deployment.yaml

```diff
@@ -44,51 +47,55 @@ spec:
       {{- end }}
       {{- if .Values.pgbouncer.podSecurityContext.enabled }}
       securityContext: {{- omit .Values.pgbouncer.podSecurityContext "enabled" | toYaml | nindent 8 }}
-      {{- end }}     
+      {{- end }}
       containers:
       - name: {{ .Chart.Name }}-pgbouncer
-      {{- if .Values.pgbouncer.image }}
-        image: "{{ default "edoburu/pgbouncer" .Values.pgbouncer.image.repository }}:{{ default "1.12.0" .Values.pgbouncer.image.tag }}"
-        imagePullPolicy: {{ default "IfNotPresent" .Values.pgbouncer.image.pullPolicy }}
-      {{- else }}
-        image: "edoburu/pgbouncer:1.12.0"
-        imagePullPolicy: "IfNotPresent"
-      {{- end }}
+        image: "{{ .Values.pgbouncer.image.repository }}:{{ .Values.pgbouncer.image.tag }}"
+        imagePullPolicy: {{ .Values.pgbouncer.image.pullPolicy }}
         ports:
         - containerPort: 6543
         env:
-        - name: DB_USER
+        - name: POSTGRESQL_USERNAME
           value: {{ include "posthog.postgresql.username" . }}
-        - name: DB_PASSWORD
+        - name: POSTGRESQL_PASSWORD
           valueFrom:
             secretKeyRef:
               name: {{ include "posthog.postgresql.secretName" . }}
               key: {{ include "posthog.postgresql.secretPasswordKey" . }}
-        - name: DB_HOST
+        - name: POSTGRESQL_HOST
           value: {{ include "posthog.postgresql.host" . }}
-        - name: DB_PORT
+        - name: POSTGRESQL_PORT
           value: {{ include "posthog.postgresql.port" . | quote }}
-        - name: DB_NAME
+        - name: POSTGRESQL_DATABASE
+          value: {{ include "posthog.postgresql.database" . }}
+        - name: PGBOUNCER_DATABASE
           value: {{ include "posthog.postgresql.database" . }}
-        - name: LISTEN_PORT
-          value: "6543"
-        - name: MAX_CLIENT_CONN
-          value: "1000"
-        - name: POOL_MODE
-          value: "transaction"
 {{- if .Values.pgbouncer.env }}
 {{ toYaml .Values.pgbouncer.env | indent 8 }}
 {{- end }}
         readinessProbe:
           tcpSocket:
             port: 6543
```

Comment:
We haven't changed the readiness probe, do you know why it wasn't working previously as expected?

Reply:
`initialDelaySeconds` is now 10 from the initial 5
`periodSeconds` is now 5 from the initial 10

I don't think we had problems before but this is a less strict config.